docs/solrini.md

```diff
@@ -98,6 +99,54 @@ Make sure `core18` collection is created and `/path/to/WashingtonPost` is update
 Solrini has also been verified to work with the [MS MARCO Passage Retrieval Corpus](experiments-msmarco-passage.md).
 There should be no major issues with other collections that are supported by Anserini, but we have not tested them.
 
+## Solr with Prebuilt Lucene Index
+
+Solr can be considered a front-end for Lucene, and it is entirely possible for Solr to read prebuilt Lucene indexes. 
+To achieve this, some housekeeping are required. The following uses [robust04](regressions-robust04.md) as an example.
```

Comment:
We adopt the convention of each sentence on a line; here and below.
robust04 -> Roboust04